core/src/main/scala/org/broadinstitute/dsde/rawls/metrics/InstrumentationDirectives.scala

```diff
@@ -38,20 +44,40 @@ trait InstrumentationDirectives extends RawlsInstrumented {
   private val redactMethodConfigs =
     (Slash ~ "api").? / "workspaces" / Segment / Segment / "methodconfigs" / Segment / (Segment ~ SegmentIgnore.repeat(separator = Slash))
 
+  private val redactGenomicsOperations =
+    (Slash ~ "api").? / "workspaces" / Segment / Segment / "genomics" / "operations" / Segment
+
   private val redactWorkspaceNames =
     (Slash ~ "api").? / "workspaces" / (!"entities" ~ Segment) / (Segment ~ SegmentIgnore.repeat(separator = Slash))
 
+  private val redactAdminBilling =
+    (Slash ~ "admin").? / "billing" / Segment / SegmentIgnore / Segment
+
+  private val redactAdminAllUserReadAccess =
+    (Slash ~ "admin").? / "allUserReadAccess" / Segment / Segment
+
+  private val redactNotifications =
+    (Slash ~ "api").? / "notifications" / "workspace" / Segment / Segment
+
+
   // Strip out unique IDs from metrics by providing a redactedUriExpansion
   override protected val UriExpansion: Expansion[Uri] = RawlsExpansion.redactedUriExpansion(
-    redactBillingProject | redactBillingProjectRoleEmail | redactUserGroup | redactUserGroupRoleEmail | redactWorkflowIds
-      | redactSubmissionIds | redactEntityIds | redactMethodConfigs | redactWorkspaceNames
+    redactBillingProject | redactBillingProjectRoleEmail | redactUserGroup | redactUserGroupRoleEmail | redactGroupAndUser | redactGroups
+      | redactWorkflowIds | redactSubmissionIds | redactEntityIds | redactMethodConfigs | redactGenomicsOperations | redactWorkspaceNames
+      | redactAdminBilling | redactAdminAllUserReadAccess | redactNotifications
   )
 
+  private lazy val globalRequestCounter = ExpandedMetricBuilder.empty.asCounter("request")
```

Comment:
I thought it would be useful to have global counters of Rawls requests/latency, in addition to the per-URI counters/timers.